web/client-api/src/main/java/io/deephaven/web/client/api/DatabarFormatBuilder.java

```diff
@@ -0,0 +1,75 @@
+package io.deephaven.web.client.api;
+
+public class DatabarFormatBuilder {
```

Comment:
I can change it back. Some of the props that use double like min, max, and opacity already have default values so null shouldn't be possible. Since value and marker prop are obtained from another column it could be null

`public DataBarFormat(double min, double max, double value, String axis, String positiveColor, String negativeColor,
            String valuePlacement, String direction, double opacity, double marker, String markerColor)`